topics/microbiome/tutorials/visualisation-ampvis/tutorial.md

```diff
@@ -0,0 +1,260 @@
+---
+layout: tutorial_hands_on
+
+title: Development of statistical analysis and visualization workflows for metagenomic
+  amplicon data using the Galaxy framework
+level: Intermediate
+zenodo_link: https://zenodo.org/records/11281381
+questions:
+- If we generated amplicon data, how can we analyse it with Galaxy ampvis2 tools?
+- How can we visualise amplicon data by using heatmap, ordination plot, boxplot, rarefraction curve or timeseries?
+objectives:
+- use heatmap workflow to analyse and visualise amplicon data
+- or use ordination plot, or boxplot, or rarefraction curve, or timeseries
+- use grouped data or grouped data with facets
+time_estimation: 3H
+key_points:
+- using different visualisation methods can present data from other points of view
+- with enough metadata the data can be visualised w.r.t. groups and facets 
+contributors:
+- lenaarenot
+- contributor2
+
+---
+
+
+
+General introduction about the topic and then an introduction of the
```

Comment:
would be nice if you can add some general introduction adapted from here: https://kasperskytte.github.io/ampvis2/articles/ampvis2.html and https://www.biorxiv.org/content/10.1101/299537v1